specification/record-locator/producer.yaml

```diff
@@ -1219,7 +1221,7 @@ components:
             - DocumentReference
         id:
           type: string
-          pattern: "[A-Za-z0-9\\-\\.]{1,64}"
+          pattern: "^(?=.{1,64}$)[A-Za-z0-9\\.]+-[A-Za-z0-9]+[A-Za-z0-9\\_\\-]*$"
```

Comment:
this doesn't look like it's related to the ticket?

Reply:
Reverted this change and opened https://nhsd-jira.digital.nhs.uk/browse/SPINECLI-1599